recipe/meta.yaml

```diff
@@ -11,18 +11,18 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - antlr ==4.8
   host:
-    - python >=3.6,<3.9
+    - python >=3.6,<3.10
     - pip
   run:
-    - python >=3.6,<3.9
-    - omegaconf >=2.0.2
+    - python >=3.6,<3.10
+    - omegaconf >=2.0.5,<2.1
     - antlr-python-runtime ==4.8
     - importlib_resources
```

Comment:
Can you add a comment this is only needed in Python < 3.9?
At some point we will be able to remove it.